### ISSUE TITLE
React-Native 0.70.6 Not Picking up RNSecureStorage.podspec

### ISSUE THREAD

**Issue by DaniShalash** [talut/rn-secure-storage#71]
It looks like React-Native 0.70.6 is not picking up `RNSecureStorage.podspec` which is placed under `ios` folder in this library, causing a `Type Error` when we use the library on JS side.

I'm not aware if prior versions of React-Native have the same issue because I jumped directly from 0.67.5 to 0.70.6.

I have solved the issue by moving `RNSecureStorage.podspec` back to the root of the repo, and modify the `license` attribute since it broke.

For anyone struggling from the same issue:

1. Delete `node_modules/ios/RNSecureStorage.podspec`.
2. Create a new file 'node_modules/RNSecureStorage.podspec'.
3. Copy paste the below config inside of it.
4. Run in terminal `npx patch-package rn-secure-storage` (Read more about [patch-package](patch-package))

```
require 'json'

package = JSON.parse(File.read(File.join(__dir__, 'package.json')))

Pod::Spec.new do |s|
  s.name         = "RNSecureStorage"
  s.version      = package["version"]
  s.summary      = "RNSecureStorage"
  s.description  = package["description"]
  s.homepage     = package["homepage"]
  s.license      = { :type => package["license"], :file => "./LICENSE.md" }
  s.author       = { "Talut Tasgiran" => "info@taluttasgiran.com.tr" }
  s.platform     = :ios, "7.0"
  s.source       = { :git => "https://github.com/talut/rn-secure-storage.git", :tag => "v#{s.version}" }
  s.source_files  = "**/*.{h,m}"
  s.requires_arc = true

  s.dependency "React"

end
```

I'll submit a PR in this couple of days as well.

**Comment by Martinocom** [comment]
Just a little note: path are missing the library name. It should be:

- Delete `node_modules/rn-secure-storage/ios/RNSecureStorage.podspec`
- Create new file `node_modules/rn-secure-storage/RNSecureStorage.podspec`

Installing Pods (when I was thinking all pods are getting installed), after the fix this appeared:
```
Downloading dependencies
Installing RNSecureStorage (2.0.8) <== Wow
Generating Pods project
```

Thanks!

**Comment by talut** [comment]
I just released v3.0.0 I hope it'll solve your problems.

Please check the new version and Readme file. It also includes a few new features as well.
